AppinfoJsonFile.js

```diff
@@ -210,33 +211,42 @@ AppinfoJsonFile.prototype.getTranslationSet = function() {
 // we don't localize or write appinfo.json source files
 AppinfoJsonFile.prototype.write = function() {};
 
-
 /**
  * Return the location on disk where the version of this file localized
  * for the given locale should be written.
  * @param {String] locale the locale spec for the target locale
  * @returns {String} the localized path name
  */
 AppinfoJsonFile.prototype.getLocalizedPath = function(locale) {
-    var rootPath = path.join(this.project.target, this.project.getResourceDirs("json")[0] || ".");
-    var fullPath = "";
+    var mapping = this.mapping || this.type.getMappings(this.pathName || "") || this.type.getDefaultMapping();
+    
     var rootLocale = "en-US";
-
     var splitLocale = locale.split("-");
     this.baseLocale = Utils.isBaseLocale(locale);
+    var resDir = this.project.getResourceDirs("json")[0] || ".";
+    var lo = locale;
 
     if (this.baseLocale) {
         if (locale !== rootLocale) {
-            fullPath = splitLocale[0];
-        }
-    } else {
-        for (var i=0; i < splitLocale.length; i++) {
-            fullPath += "/"+ splitLocale[i];
+            lo = splitLocale[0];
         }
     }
-    return path.join(rootPath, fullPath);
+    var path = this.API.utils.formatPath(mapping.template, {
+        sourcepath: this.pathName,
+        resourceDir: resDir,
```

Comment:
Doesn't this need to pass in the project and type info?

Reply:
I kept the previous way by transferring resources directly within the second parameter's object. and this plugin chooses that way.  (checking parameter's resourceDir first)
https://github.com/iLib-js/loctool/blob/getFormatPath/lib/utils.js#L1939